web/src/components/ThemeContainer.tsx

```diff
@@ -33,25 +34,30 @@ export const BREAKPOINTS = {
 const createTheme = (
   themeType: 'light' | 'contrast',
   contentDirection: UiDirectionType,
-): Omit<Theme, 'toggleTheme'> => ({
-  ...(themeType === 'contrast' ? buildConfig().legacyContrastTheme : buildConfig().legacyLightTheme),
-  contentDirection,
-  isContrastTheme: themeType === 'contrast',
-  ...createMuiTheme({
-    breakpoints: {
-      values: BREAKPOINTS,
-    },
-    direction: contentDirection,
-    colorSchemes: {
-      light: buildConfig().lightTheme,
-      dark: buildConfig().darkTheme,
-    },
-    typography: buildConfig().typography,
-    palette: {
-      mode: themeType === 'contrast' ? 'dark' : 'light',
-    },
-  }),
-})
+): Omit<Theme, 'toggleTheme'> => {
+  const isContrast = themeType === 'contrast'
+  const commonPalette = isContrast ? commonDarkColors : commonLightColors
+
+  return {
+    ...(isContrast ? buildConfig().legacyContrastTheme : buildConfig().legacyLightTheme),
+    contentDirection,
+    isContrastTheme: isContrast,
+    ...createMuiTheme({
+      breakpoints: {
+        values: BREAKPOINTS,
+      },
+      direction: contentDirection,
+      colorSchemes: {
+        light: buildConfig().lightTheme,
+        dark: buildConfig().darkTheme,
+      },
+      typography: buildConfig().typography,
+      palette: {
+        ...commonPalette,
```

Comment:
I needed to add this to have an access to commonColors from theme/colors.tsx. I am not sure whether it is the right way to do this...

Reply:
What color do you need access for, i.e. what exactly does not work otherwise? In theory everything should already be available.